lib/test/broker.go

```diff
@@ -32,6 +32,13 @@ func BrokerCreate(r *KnRunResultCollector, name string) {
 	assert.Check(r.T(), util.ContainsAllIgnoreCase(out.Stdout, "Broker", name, "created", "namespace", r.KnTest().Kn().Namespace()))
 }
 
+// BrokerCreateWithClass creates a broker with the given name and class.
```

Comment:
Why is these not named `CreateBroker…`?

Reply:
The convention in this file is BrokerCreate, BrokerDelete etc so I am just following that.

Reply:
The convention was already established in the file, not change by this PR. I propose to keep it as it and if we want to change it do it in a separate move.